scripts/configure-hypervisor.sh

```diff
@@ -268,12 +268,12 @@ generate_qemu_options()
 	fi
 
 	# Not required as "-uuid ..." is always passed to the qemu binary
-	qemu_options+=(size:--disable-uuid)
-
 	# Disable debug
 	case "$arch" in
-		aarch64) ;;
+		aarch64) qemu_options+=(size:--disable-uuid)
```

Comment:
woot!, this PR if for `ppc64le` or `aarch64`?

Reply:
cc @Pennyzct @Weichen81

Reply:
@devimc : qemu_options+=(size:--disable-uuid) was for all archs earlier and I wanted to be that way except for ppc64le. Hence the change.